include/fmt/chrono.h

```diff
@@ -22,6 +22,13 @@
 
 FMT_BEGIN_NAMESPACE
 
+// check if std::chrono::utc_timestamp is available
+#ifndef FMT_USE_UTC_TIME
+#  ifdef __cpp_lib_chrono
```

Comment:
I think this can be simplified:

```
#if !defined(FMT_USE_UTC_TIME) && defined(__cpp_lib_chrono)
```

Reply:
But I forgot `define FMT_USE_UTC_TIME 0` in the else-branch of `ifdef __cpp_lib_chrono`